src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+//TODO: Decide about i18n solution or simply a JSON file where we can load translations.
+
+const TRANSLATIONS = {
+  SHIP_TO_ME: 'Ship to Me',
+  ARRIVES: 'Arrives'
+};
+
+const Container = styled.div`
```

Comment:
I would go with `Wrapper` naming, since we have already couple of component like that.